.github/workflows/kind-e2e.yaml

```diff
@@ -31,11 +31,11 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         pipelines-release:
```

Comment:
Can you add a comment that this should follow the list of versions from [here](https://github.com/tektoncd/pipeline/blob/main/releases.md#release)?

Reply:
Yeah make sense, will add 👍🏻